src/main/java/bwapi/UnitType.java

```diff
@@ -252,7 +252,7 @@ public enum UnitType {
         Arrays.stream(UnitType.values()).forEach(v -> idToEnum[v.id] = v);
     }
 
-    final int id;
+    public final int id;
```

Comment:
If we expose `id`, I think it's best to add it to all Enums for consistency reasons:
 - Race
 - BulletType
 - CommandType
 - CoordinateType
 - EventType
 - GameType
 - Latency
 - Order
 - PlayerType
 - ShapeType
 - TechType
 - UnitType
 - UpgradeType